satpy/readers/viirs_l1b.py

```diff
@@ -34,6 +34,21 @@ def _parse_datetime(self, datestr):
         """Parse datetime."""
         return dt.datetime.strptime(datestr, "%Y-%m-%dT%H:%M:%S.000Z")
 
+    @property
+    def day_night_flag(self):
+        """Get the day/night flag."""
+        return self["/attr/DayNightFlag"]
+
+    @property
+    def start_direction(self):
+        """Get the swath start direction flag."""
+        return self["/attr/startDirection"]
+
+    @property
+    def end_direction(self):
+        """Get the swath end direction flag."""
+        return self["/attr/endDirection"]
+
```

Comment:
I don't see a reason for these to be properties on the class. Could you change this to just access `self["/attr/endDirection"]` (and the others) directly in the `get_metadata` method?